boards/common/remote/Makefile.include

```diff
@@ -5,28 +5,35 @@ export CPU_MODEL =  cc2538sf53
 # define the default flash-tool
 export PROGRAMMER ?= cc2538-bsl
 
+# define the default reset-tool
+RESETER ?= jlink
```

Comment:
I would  not make jlink the default, because the pinout is not present by default but USB is. If you need that for testing on IoT lab or similar just set the environmental variable on that node accordingly.

Reply:
Well the point here is to make automated testing on Murdock slaves, where some testing boards are attached. I don't know if we can pass environmental variables to the command for flashing (e.g. `make flash-only`). @kaspar030 is that possible?